routes/transaction.go

```diff
@@ -98,6 +98,37 @@ func (fes *APIServer) SubmitTransaction(ww http.ResponseWriter, req *http.Reques
 		return
 	}
 
+	// If this is a creator coin transaction and this node is configured for Jumio, we update global state user metadata to say user has purchased CC.
+	if txn.TxnMeta.GetTxnType() == lib.TxnTypeCreatorCoin && txn.TxnMeta.(*lib.CreatorCoinMetadataa).OperationType == lib.CreatorCoinOperationTypeBuy {
+		var userMetadata *UserMetadata
+		userMetadata, err = fes.getUserMetadataFromGlobalStateByPublicKeyBytes(txn.PublicKey)
+		if err != nil {
+			_AddBadRequestError(ww, fmt.Sprintf("SubmitTransactionRequest: Problem getting usermetadata from global state for basic transfer: %v", err))
+			return
+		}
+		if !userMetadata.HasPurchasedCreatorCoin {
+			userMetadata.HasPurchasedCreatorCoin = true
+			if err = fes.putUserMetadataInGlobalState(userMetadata); err != nil {
+				_AddBadRequestError(ww, fmt.Sprintf("SubmitTransactionRequest: Problem updating HasPurchasedCreatorCoin in global state user metadata: %v", err))
+				return
+			}
+		}
+	}
+
+	// If this is a basic transfer, we check if user has purchased CC (if this node is configured for Jumio or Twilio)
+	if txn.TxnMeta.GetTxnType() == lib.TxnTypeBasicTransfer && (fes.IsConfiguredForJumio() || fes.Twilio != nil){
```

Comment:
Why do we care if the node is configured for jumio or twilio? Should we set it regardless?

Reply:
it would make a node run faster if they didn't fetch the user metadata for every basic transfer

Reply:
Basically, the usermetadata would never have a phone number that isn't the empty string nor would jumio verified ever be true, so it saves a trip to global state.